SPECS/libsolv/libsolv.spec

```diff
@@ -1,10 +1,10 @@
 Summary:        A free package dependency solver
 Name:           libsolv
-Version:        0.7.24
+Version:        0.7.27
 Release:        1%{?dist}
 License:        BSD
 URL:            https://github.com/openSUSE/libsolv
-Source0:        https://github.com/openSUSE/libsolv/archive/%{version}/%{name}-%{version}.tar.gz
+Source0:        https://src.fedoraproject.org/lookaside/pkgs/libsolv/%{name}-%{version}.tar.gz/sha512/5f4de944114cebbcf35a993e5f969c3b7bf3b8dbc5dabab38014bd2d1ab83b524d1ed5b7fbed4b2f751c1f8043613f9e88cf96b14b121362655af1805d238b60/%{name}-%{version}.tar.gz
```

Comment:
I think if we look at the tags, i could find a tar.gz url :)
URL: https://github.com/openSUSE/libsolv/archive/refs/tags/0.7.28.tar.gz

Reply:
Thanks and actually its a newer revision than the one I have right now I have 0.7.27 and seems that 0.7.28 is out I think I'll abandon this review and start a new one with the latest revision